README.md

```diff
@@ -47,9 +47,29 @@ var req = request(options, function callback(err, response, body) {
 ```javascript
 var requestPromise = require("client-request/promise")
 
-requestPromise(options).then(function (body) {
-  console.log(body)
-})
+requestPromise(options).then(function (result) {
+  if (result.response.statusCode === 201) {
+    console.log(result.response.headers.location);
+  } else {
+    console.log(result.body);
```

Comment:
Some semicolons got into the README examples, which mostly only matters for consistency; probably best to have consistent style throughout.